src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -144,18 +150,30 @@ public virtual AuthenticatedEncryptionProvider CreateAuthenticatedEncryptionProv
         /// </summary>
         /// <param name="key">the <see cref="SecurityKey"/> to use.</param>
         /// <param name="algorithm">the algorithm to use.</param>
-        /// <returns>an instance of <see cref="KeyWrapProvider"/></returns>
-        /// <exception cref="ArgumentNullException">'key' is null.</exception>
-        /// <exception cref="ArgumentNullException">'algorithm' is null or empty.</exception>
-        /// <exception cref="ArgumentException">If <see cref="SecurityKey"/> and algorithm pair are not supported.</exception>
+        /// <exception cref="ArgumentNullException">thrown if <paramref name="key"/> is null.</exception>
+        /// <exception cref="ArgumentNullException">thrown if <paramref name="algorithm"/> is null or empty.</exception>
+        /// <exception cref="NotSupportedException">thrown if If <see cref="SecurityKey"/> and algorithm pair are not supported.</exception>
```

Comment:
nit: extra "if"

Reply:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/dd4056bb650b8ec7a48d45c0e62e6454da1c7736